File: dayTwo/javascript/level 1/TaskTen.js
const prompt = require("prompt-sync")()

let first_number = +prompt("Enter first number: ")

let second_number = +prompt("Enter second number: ")

let third_number = +prompt("Enter third number: ")

let addition = first_number + second_number + third_number

let average = addition/3

console.log(average)

File: dayTwo/java/level 1/TaskTen.java
import java.util.Scanner;
public class TaskTen {

    public static void main(String[] args) {
    
        Scanner userInput = new Scanner(System.in);
        
        System.out.print("Enter the first number: ");
        int firstNumber = userInput.nextInt();

        System.out.print("Enter the second number: ");
        int secondNumber = userInput.nextInt();

        System.out.print("Enter the third number: ");
        int thirdNumber = userInput.nextInt();

        int sum = firstNumber + secondNumber + thirdNumber;

        double average = sum/3;

        System.out.print(average);

    }

}
